### PR TITLE
Rebuild v0.41.0:

### PR DESCRIPTION
**Pull request by akabanovs**
- build number bumped up.
 - build script updated with --no-build-isolation flag.
 - 'about' section updated.

### CODE REVIEW

File: recipe/meta.yaml
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:27]: is make actually needed ?, otherwise LGTM

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml
Comment (akabanovs) [reply recipe/meta.yaml:27]: Actually, not sure why compiler is needed too. This is a rebuild so I did not pay attention to that.

Comment (akabanovs) [reply recipe/meta.yaml:27]: but now it failed, so it needs it :-)

**Review by ViridianMelody — APPROVED** [review]
LGTM pending successful win-64 build.

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**
